website/docs/language/import/index.mdx

```diff
@@ -0,0 +1,188 @@
+---
+page_title: Import - Configuration Language
+description: >-
+  Import resources into Terraform to bring them under Terraform's management by writing `import` blocks in code.
+---
+
+# Import
+
+-> **Note:** Import blocks are only available in Terraform v1.5.0 and later.
+
+~> **Experimental:** Import blocks are an experimental Terraform feature. While we do not expect to make backwards-incompatible changes to their syntax, processing of imports during the plan, generating configuration, and the `-generate-config-out` flag may change in future releases.
+
+Terraform can import existing infrastructure resources, which brings them under Terraform's management. Once you have imported a resource, it will be present in the Terraform configuration and state, just like a resource you had created via Terraform. You can then manage it like any other resource, updating its attributes and destroying it as part of a normal resource lifecycle.
+An `import` block records that Terraform imported the resource, that it was not originally created due to this Terraform configuration. Considered another way, it records the user's intent that an import be added to Terraform's plan graph.
+
+## `import` block syntax
+
+The `import` block can appear in any Terraform configuration file. A common pattern is to create an `imports.tf` file, or to place each `import` block next to the `resource` block it will import into.
+
+```hcl
+import {
+  to = aws_instance.example
+  id = "i-abcd1234"
+}
+```
+
+The above `import` block defines an import of the AWS instance with ID "i-abcd1234" into the resource address `aws_instance.example` in the root module.
+
+The `import` block has the following arguments:
+ - `to` - The resource instance address targeted by this import.
+ - `id` - The import ID of the resource. String.
+ - `provider` (optional) - A custom resource provider. See [The Resource provider Meta-Argument](/terraform/language/meta-arguments/resource-provider).
+
+## Planning and applying an import
+
+The `import` block, like all Terraform configuration blocks, is processed during the `terraform plan` operation, which outputs a list of changes Terraform will make if you proceed to apply. Along with create, update, remove, and move, import is an action Terraform can perform on a resource. The resource is only added to the state during `terraform apply`.
+
+### 1. Add the `import` block to configuration
```

Comment:
I feel like these steps belong in a tutorial walkthough rather than docs; I suggest we remove these steps in favor of a bulleted list explaining the steps. 

```suggestion
You can use `import` blocks by doing the following:
1. Define an `import` block for the resource(s).
1. Add a `resource` block in your configuration.
1. Run `terraform plan` to see how Terraform plans to import the resource(s).
1. Apply the configuration and have Terraform import resources and update your Terraform state.

> **Hands-on:** Try the [State Import](/terraform/tutorials/state/state-import?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
```

Reply:
Also, do we need to add a step about refreshing state in this list?

Reply:
No, the state is refreshed automatically during plan.